src/Plugin/OgFields/AccessField.php

```diff
@@ -22,61 +19,60 @@ class AccessField extends OgFieldBase implements OgFieldsInterface {
   /**
    * {@inheritdoc}
    */
-  public function fieldDefinition() {
-    return FieldStorageConfig::create(array(
-      'field_name' => OG_DEFAULT_ACCESS_FIELD,
-      'entity_type' => $this->getEntityType(),
-      'type' => 'list_integer',
-      'cardinality' => 1,
-      'settings' => array(
-        'allowed_values' => array(
+  public function getFieldStorageConfigBaseDefinition(array $values = array()) {
+    $values = [
+      'cardinality' => FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED,
+      'settings' => [
+        'allowed_values' => [
           0 => 'Use default roles and permissions',
           1 => 'Override default roles and permissions',
-        ),
+        ],
         'allowed_values_function' => '',
-      ),
-    ));
+      ],
+      'type' => 'list_integer',
+    ];
+
+    return parent::getFieldStorageConfigBaseDefinition($values);
```

Comment:
This is not covered by tests. PHPStorm indicates that `FieldStorageDefinitionInterface` is an undefined class, so a use statement for this should be added.